scripts/forge_auto.lua

```diff
@@ -1031,7 +1031,7 @@ end
 
 function downArrow()
   srReadScreen();
-  downPin = srFindImage("Fishing/Menu_DownArrow.png");
+  downPin = srFindImage("menu_DownArrow.png");

```

Comment:
Was this change intentional?  Only similarly named image on disk is `fishing/Menu_DownArrow`?